apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.ts

```diff
@@ -12,8 +12,8 @@ import {
 export class ItemShowcaseComponent {
   properties: ApiDescriptionProperty[] = [
     {
-      name: 'Selectable',
-      description: 'Make item selectable',
+      name: 'selectable',
+      description: 'If `true`, a native button tag will be rendered and the item will be tappable.',
```

Comment:
Consider adding a bit of context
```suggestion
      description: 'If `true`, a native button tag will be rendered under the hood and the item will become interactive.',
```